src/popup/components/InputField.vue

```diff
@@ -142,9 +147,12 @@ export default defineComponent({
       },
       default: null,
     },
+    help: {
+      type: Object as PropType<{ title: string; msg: string; fullscreen: Boolean }>,
+      default: null,
+    },
     readonly: Boolean,
     showHelp: Boolean,
-    showMessageHelp: Boolean,
```

Comment:
Nice finding as it was not handled within the component. But please take a look on 2 components that was setting this prop: `TransferSendRecipient` and `AppsBrowser`.